### PR TITLE
feat!: support Angular 15

### PR DESCRIPTION
**Pull request by johnhwhite**
[AB#2398383](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2398383)

### CODE REVIEW

File: tsconfig.base.json [hunk 1]
```diff
@@ -19,6 +20,7 @@
     "noFallthroughCasesInSwitch": true,
     "baseUrl": ".",
     "paths": {
+      "@skyux-sdk/tools": ["libs/sdk/tools/src/index.ts"],
```
Comment (Blackbaud-SteveBrush) [inline tsconfig.base.json]: Eh?

File: libs/stache/src/lib/modules/router/route.service.spec.ts [hunk 2]
```diff
@@ -270,11 +270,11 @@ describe('StacheRouteService', () => {
 
   it('should not unset the active routes on NavigationEnd', async () => {
     await setupTest();
-    (router as any).events = observableOf(new NavigationEnd(0, '', ''));
     spyOn(StacheRouteService.prototype, 'clearActiveRoutes');
+    await router.navigate(['a']);
     expect(
       StacheRouteService.prototype.clearActiveRoutes
-    ).not.toHaveBeenCalled();
+    ).toHaveBeenCalledTimes(1);
```
Comment (johnhwhite) [inline libs/stache/src/lib/modules/router/route.service.spec.ts:277]: Angular changed `router.events` to be read-only.

File: libs/stache/tsconfig.spec.json [hunk 3]
```diff
@@ -2,7 +2,8 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "outDir": "../../dist/out-tsc",
-    "types": ["jasmine", "node"]
+    "types": ["jasmine", "node"],
+    "resolveJsonModule": true
```
[the remark this answers is not in the document]
Comment (johnhwhite) [reply libs/stache/tsconfig.spec.json]: Removed them. I was looking at pulling the `@skyux/core` version from `package.json` but it crossed boundaries and I abandoned the approach.

File: libs/stache/schematics/ng-add/ng-add.schematic.spec.ts [hunk 4]
```diff
@@ -3,11 +3,11 @@ import {
   UnitTestTree,
 } from '@angular-devkit/schematics/testing';
 
-import path from 'path';
+import { joinPathFragments } from 'nx/src/utils/path';
 
 import { createTestLibrary } from '../testing/scaffold';
 
-const COLLECTION_PATH = path.resolve(__dirname, '../../collection.json');
+const COLLECTION_PATH = joinPathFragments(__dirname, '../../collection.json');
```
[the remark this answers is not in the document]
Comment (johnhwhite) [reply libs/stache/schematics/ng-add/ng-add.schematic.spec.ts]: Switched.

File: package.json [hunk 5]
```diff
@@ -81,33 +81,34 @@
...
     "@types/cross-spawn": "6.0.2",
     "@types/fs-extra": "11.0.1",
     "@types/jasmine": "4.0.3",
-    "@types/lodash.get": "4.4.2",
+    "@types/lodash.get": "4.4.7",
     "@types/node": "16.11.7",
     "@types/prismjs": "1.26.0",
     "@typescript-eslint/eslint-plugin": "5.51.0",
     "@typescript-eslint/parser": "5.51.0",
+    "axe-core": "^4.6.3",
```
[the remark this answers is not in the document]
Comment (johnhwhite) [reply package.json]: Updated.

File: tsconfig.base.json (hunk 1 above)
Comment (johnhwhite) [reply tsconfig.base.json]: Not needed, removed it.

File: libs/stache/src/lib/modules/router/route.service.spec.ts (hunk 2 above)
Comment (Blackbaud-SteveBrush) [reply libs/stache/src/lib/modules/router/route.service.spec.ts:277]: 👍

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**